controller/auth_controller.js

```diff
@@ -88,7 +88,7 @@ const signin = async (req, res) => {
     if (!identifier || !password) {
       return res
         .status(400)
-        .json({ message: "Email and Password are required" });
+        .json({ message: "identifier and password are required" });
```

Comment:
user might not know what "identifier" is, 

is it possible to have something that is more explanatory if we must remove email